index.d.ts

```diff
@@ -0,0 +1,45 @@
+/// <reference types="node" />
+
+import { FastifyPlugin, FastifyRequest } from 'fastify';
+
+declare module 'fastify' {
+  interface FastifyInstance {
+    csrfProtection(req: FastifyRequest, reply: FastifyReply, done: any): any;
```

Comment:
`done` should at least be a function or something like `() => {}`.

The return type should be `void` if it doesn't return anything (or another appropriate data type)